src/VirtualClient/VirtualClient.Contracts/VirtualClientComponent.cs

```diff
@@ -121,13 +121,7 @@ public string ContentPathFormat
         /// Parameter defines the content path format/structure using a template to use when uploading content
         /// to target storage resources. When not defined the 'Default' structure is used.
```

Comment:
In the comment you mentioned default, it would be good to also add default to comment if it's not in the code